Meadowlark-js/packages/meadowlark-authz-server/src/Utility.ts

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+export function clientIdFrom(pathExpression: RegExp, path: string): string {
```

Comment:
I pulled this into it's own file since it's useful for determining id's both updates and resets. I am not great with RegEx so we needed different expressions for update and reset (because of the extra 'reset' path component) if there is a better RegEx that can handle both at once, let me know.